### PR TITLE
Fix anchors and scrolling issues

### LINKED ISSUE #111: hashtags references are ignored on the first load
This URL doesn't scroll down to the `wb_robot_get_device` section, as it should:
https://www.cyberbotics.com/doc/reference/robot?version=fix-robot-doc#wb_robot_get_device

### LINKED ISSUE #112: Vertical scrollbar should be reset to the top when changing page
When you visit https://www.cyberbotics.com/doc/reference/robot and scroll down and then you click on the Camera section, the camera documentation is displayed, but the vertical scrollbar is not reset to the top as the user would expect. That looks wrong.

### PR DESCRIPTION
**Pull request by fabienrohrer**
- [x] Fix #112 
- [x] Remove the weird and problematic local scrolling offset
- [x] Fix #111 
- [x] Upload `viewer.js` to FTP

### CODE REVIEW

File: js/viewer.js [hunk 1]
```diff
@@ -183,8 +183,6 @@ function applyAnchor() {
         anchors[0].scrollIntoView(true);
         if (!local)
           window.scrollBy(0, -46); // 46 is the height of the header of Cyberbotics web page
-        else
-          window.scrollBy(0, 180); // manual adjustment for the off-line version
```
Comment (fabienrohrer) [inline js/viewer.js]: I think I understood the issue about this.

It's related to images, if the server is too slow to provide them, the scrolling is taking into account only the text, so the anchors above can be delayed (I can reproduce this systematically on my VM-Windows, and sometimes after clearing the cache on my Mac).
The best solution seems to scroll after the images are actually loaded.

**Comment by fabienrohrer** [comment]
👍

**Comment by omichel** [comment]
:eyes:

**Comment by omichel** [comment]
:+1:

**Pull request merged**